### PR TITLE
Using kubernetes-mixin as a base, add a README.

### PR DESCRIPTION
**Pull request by tomwilkie**
/cc @metalmatze @brancz

### CODE REVIEW

**Comment by brancz** [comment]
The design as a pdf seems kind of odd, why not markdown?

**Comment by brancz** [comment]
I just cross read, but this lgtm in my experience this will improve over time anyways as the community contributes things that are unclear where we are likely too close to the concept to realize.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+1. Use kube-prometheus to deploy this mixin.
+
+## Generate config files
+
+You can manually generate the alerts, dashboards and rules files, but first you
+must install some tools:
+
+```
+$ go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+$ brew install jsonnet
```
Comment (metalmatze) [inline README.md:34]: IMHO we should not only show how to brew install here

Comment (metalmatze) [inline README.md:34]: Just found that nobody really has official support yet. Maybe linking to AUR is ok, but also not as important then.
```suggestion
# macOS
$ brew install jsonnet
# Archlinux AUR 
$ yay -S jsonnet
```

**Review by metalmatze — APPROVED** [review]
LGTM
I'm also not sure about the PDF file though. Maybe markdown with a link to the original is good enough.

**Comment by tomwilkie** [comment]
PDF because its a google doc, and there is no export as markdown.  Plus that design is kinda immutable now - future changes should be new designs IMO.

**Pull request merged**